src/Resources/config/controllers.php

```diff
@@ -36,5 +38,15 @@
                 new ReferenceConfigurator('sonata.media.pool'),
             ])
             ->tag('container.service_subscriber')
+            ->call('setContainer', [new ReferenceConfigurator(ContainerInterface::class)])
+
+        ->set(MediaAdminController::class)
```

Comment:
assign a name to this please like above

Reply:
Name added. 
Note that I had to add also an alias.

Reply:
The `sonata.media.controller.gallery` does not have an alias.
Why the alias would be require here @jordisala1991 ?

Reply:
I don't know if it should be needed. Maybe changing the admin definition to not rely on the controller class, but the service id.

Reply:
I confirm that if I don't put this alias, the `sonata.media.controller.media.admin` service is not used. Seems related to the Route builder which deduct the controller and this controller is then instanciated by the SF Controller resolver without its correct configuration.

Reply:
You need to change both this files:

https://github.com/sonata-project/SonataMediaBundle/blob/master/src/Resources/config/doctrine_mongodb_admin.php
https://github.com/sonata-project/SonataMediaBundle/blob/master/src/Resources/config/doctrine_orm_admin.php

Concretely those lines:

https://github.com/sonata-project/SonataMediaBundle/blob/master/src/Resources/config/doctrine_mongodb_admin.php#L42
https://github.com/sonata-project/SonataMediaBundle/blob/master/src/Resources/config/doctrine_mongodb_admin.php#L68
https://github.com/sonata-project/SonataMediaBundle/blob/master/src/Resources/config/doctrine_orm_admin.php#L42
https://github.com/sonata-project/SonataMediaBundle/blob/master/src/Resources/config/doctrine_orm_admin.php#L69